File: programme/volume-of-sphere/VolumeOfSphere.java
// Write a program to compute the volume of a Sphere

import java.util.Scanner;

public class VolumeOfSphere {
	private static double computeSphereVolume(double r) {
		return r * r * r * 4 / 3 * Math.PI;
	}

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.print("Radius : ");
		double r = sc.nextDouble();
		double vol = computeSphereVolume(r);
		System.out.printf("\nVolume : %.2f", vol);
		sc.close();
	}
}

File: programme/volume-of-sphere/volume-of-sphere.js
// Write a program to compute the volume of a Sphere

"use strict";

// perform following command to run programme
// node volume-of-sphere.js

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question('Radius : ', r => {
	const v = Math.pow(Number(r), 3) * 4/3 * Math.PI;
	if (isNaN(v)) {
		console.error('Some error occured');
	} else {
		console.log(`Volume : ${v.toFixed(2).replace(/\.?0*$/,'')}`);
	}
	rl.close();
});
